tealer/detectors/can_delete.py

```diff
@@ -25,6 +25,18 @@ def _is_oncompletion_check(ins1: Instruction, ins2: Instruction) -> bool:
     return False
 
 
+def _is_application_creation_check(ins1: Instruction, ins2: Instruction) -> bool:
```

Comment:
This is copied in two detectors, maybe we can move it somewhere else? Like a instruction_utils or something like that.

I just noticed that `_is_oncompletion_check` is also really similar (aside `UpdateApplication`/`DeleteApplication`), we could also refactor this.

In the long term, having a list of simple utils for these will be useful when building detectors

Reply:
Discussed on slack: this will be done on a separate PR